scripts/benchmarks/benches/velocity.py

```diff
@@ -27,20 +28,20 @@ def download_deps(self):
         return
 
     def setup(self):
-        self.download_deps()
-
         build_path = create_build_path(self.directory, self.bench_name)
+        self.benchmark_bin = os.path.join(build_path, self.bin_name)
 
-        configure_command = [
-            "cmake",
-            f"-B {build_path}",
-            f"-S {self.code_path}",
-            f"-DCMAKE_BUILD_TYPE=Release"
-        ]
-        run(configure_command, {'CC': 'clang', 'CXX':'clang++'}, add_sycl=True)
-        run(f"cmake --build {build_path} -j", add_sycl=True)
+        if options.rebuild:
+            self.download_deps()
```

Comment:
this is not necessary, the underlying `download_untar` function will skip the file download if it exists.